SteamBot/Bot.cs

```diff
@@ -336,6 +364,24 @@ public void HandleBotCommand(string command)
             }
         }
 
+        public void HandleInput(string input)
+        {
+            consoleInput = input;
+        }
+
+        public string WaitForInput()
+        {
+            consoleInput = null;
+            while (true)
+            {
+                if (consoleInput != null)
+                {
+                    return consoleInput;
+                }
+                Thread.Sleep(5);
+            }
+        }
```

Comment:
We should probably switch to using ResetEvents wait intervals than
Thread.Sleep should we not?
On 2 Dec 2015 11:16 p.m., "Joel" <notifications@github.com> wrote:

> In SteamBot/Bot.cs
> <https://github.com/Jessecar96/SteamBot/pull/853#discussion_r46491479>:
>
> > +        {
> > +            consoleInput = input;
> > +        }
> > +
> > +        public string WaitForInput()
> > +        {
> > +            consoleInput = null;
> > +            while (true)
> > +            {
> > +                if (consoleInput != null)
> > +                {
> > +                    return consoleInput;
> > +                }
> > +                Thread.Sleep(5);
> > +            }
> > +        }
>
> No different than this:
> https://github.com/Jessecar96/SteamBot/blob/master/SteamBot/Bot.cs#L942
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Jessecar96/SteamBot/pull/853/files#r46491479>.
>